<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nBoxLineout</class>
 <widget class="QMainWindow" name="nBoxLineout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="nCustomDoublePlot" name="plot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadPref"/>
   <addaction name="actionSavePref"/>
   <addaction name="separator"/>
   <addaction name="actionSaveClipboard"/>
   <addaction name="actionSaveTxt"/>
   <addaction name="actionSavePDF"/>
   <addaction name="separator"/>
   <addaction name="actionRect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionLoadPref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//loadPref.png</normaloff>:icons//loadPref.png</iconset>
   </property>
   <property name="text">
    <string>loadPref</string>
   </property>
   <property name="toolTip">
    <string>Load Preferences</string>
   </property>
  </action>
  <action name="actionSavePref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePref.png</normaloff>:icons//savePref.png</iconset>
   </property>
   <property name="text">
    <string>savePref</string>
   </property>
   <property name="toolTip">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionSaveClipboard">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveClipboard.png</normaloff>:icons//saveClipboard.png</iconset>
   </property>
   <property name="text">
    <string>saveClipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionSaveTxt">
   <property name="icon">
    <iconset>
     <normaloff>:icons//saveTxt.png</normaloff>:icons//saveTxt.png</iconset>
   </property>
   <property name="text">
    <string>saveTxt</string>
   </property>
   <property name="toolTip">
    <string>Export TXT</string>
   </property>
  </action>
  <action name="actionSavePDF">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePDF.png</normaloff>:icons//savePDF.png</iconset>
   </property>
   <property name="text">
    <string>savePDF</string>
   </property>
   <property name="toolTip">
    <string>Export PDF or SVG</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="icon">
    <iconset>
     <normaloff>:icons//rect.png</normaloff>:icons//rect.png</iconset>
   </property>
   <property name="text">
    <string>rect</string>
   </property>
   <property name="toolTip">
    <string>Show Region</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nCustomDoublePlot</class>
   <extends>QWidget</extends>
   <header>nCustomPlots.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
